src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.IO;
+using NUnit.Framework;
+using Xamarin.ProjectTools;
+using System.Linq;
+using System.Text;
+using System.Collections.Generic;
+using System.Xml.Linq;
+
+namespace Xamarin.Android.Build.Tests
+{
+	[Parallelizable (ParallelScope.Fixtures)]
+	public class PackagingTest : BaseTest
+	{
+#pragma warning disable 414
+		static object [] ManagedSymbolsArchiveSource = new object [] {
+			//           isRelease, monoSymbolArchive, archiveShouldExists,
+			new object[] { false    , false              , false },
+			new object[] { true     , true               , true },
+			new object[] { true     , false              , false },
+		};
+#pragma warning restore 414
+
+		[Test]
+		[TestCaseSource ("ManagedSymbolsArchiveSource")]
```

Comment:
Since we have C#6 now, this should use `nameof(ManagedSymbolsArchiveSource)`. :-)